tests/parsing-tests/unit-tests.ref

```diff
@@ -92842,6 +92842,14 @@
             }
           ]
         },
+        "ios_xr_ospf" : {
+          "Red flags" : [
+            {
+              "tag" : "MISCELLANEOUS",
+              "text" : "Interface: 'Bundle-Ethernet201' contains OSPF settings, but there is no OSPF process"
```

Comment:
Just skimming this file looks like ospf process exists. Is this warning correct?